emperor/support_files/js/controller.js

```diff
@@ -66,23 +94,39 @@ EmperorController.prototype.resize = function(width, height){
   // the area we have to present the plot is smaller than the total
   var plotWidth = this.$plotSpaceId.width();
 
+  // TODO: The below will need refactoring
+  // This is addressed in issue #414
   if (this.sceneViews.length === 1) {
     this.sceneViews[0].resize(0, 0, plotWidth, this.height);
   }
   else if (this.sceneViews.length === 2) {
-    this.sceneViews[0].resize(0, 0, 0.5 * plotWidth, this.height);
-    this.sceneViews[1].resize(0.5 * plotWidth, 0, 0.5 * plotWidth, this.height);
+    this.sceneViews[0].resize(0, 0, sceneViewScale * plotWidth, this.height);
+    this.sceneViews[1].resize(sceneViewScale * plotWidth, 0,
+                              sceneViewScale * plotWidth, this.height);
   }
   else if (this.sceneViews.length === 3) {
-    this.sceneViews[0].resize(0, 0, 0.5 * plotWidth, 0.5 * this.height);
-    this.sceneViews[1].resize(0.5 * plotWidth, 0, 0.5 * plotWidth, 0.5 * this.height);
-    this.sceneViews[2].resize(0, 0.5 * this.height, plotWidth, 0.5 * this.height);
+    this.sceneViews[0].resize(0, 0,
+			      sceneViewScale * plotWidth,
```

Comment:
I think there is some weird formatting in the browser.  If you still see this problem, try looking in the github editor? They are aligned in my editor and the github editor